gallery_dl/extractor/facebook.py

```diff
@@ -0,0 +1,454 @@
+# -*- coding: utf-8 -*-
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+"""Extractors for https://www.facebook.com/"""
+
+from .common import Extractor, Message
+from .. import text, exception
+
+BASE_PATTERN = r"(?:https?://)?.*?facebook\.com"
```

Comment:
```suggestion
BASE_PATTERN = r"(?:https?://)?(?:\w+\.)?facebook\.com"
```
or something that matches whatever possible subdomains there are.